sns-book/src/domain-name/domain-find-for-owner.md

```diff
@@ -1,6 +1,28 @@
 # Get all domains of a user
 
-You can retrieve all the domain names of a user with the following `MemcmpFilter`
+If you are using `@bonfida/spl-name-service` you can use the following code:
+
+```js
+import { getAllDomains } from "@bonfida/spl-name-service";
+
+// ...
+
+const domains = await getAllDomains(connection, user);
+```
+
+The function above will return an array of 2 Public Keys. Use the toBase58() method to convert the keys into a human readable format.
+
+```js
+domains.map((domain) => domain.toBase58());
+```
+
+Another option to retrieve not only human readable Public Keys, but also the domain name in a human readable string format is to use the following function from spl-name-service library below.
```

Comment:
1. Same as above for `human readable Public Keys` it reads a bit weird

Reply:
Maybe something like, "Another option to retrieve public keys, as well as  their corresponding domain names in a string format is to use..."